packages/ng-datasource/typings/progress.data.ng.ds.d.ts

```diff
@@ -12,21 +12,29 @@ export declare class DataSourceOptions {
     mergeMode?: any;
     pageSize?: any;
     readLocal?: boolean;
+    countFnName?: string;
+}
+export interface DataResult {
```

Comment:
Confirm that tslint is running using tslint.json for NativeScript. If error with "interface" change to "class".

Reply:
I have added tslint.json in my project and it does not complain about the interface